src/hotspot/share/gc/shenandoah/shenandoahVerifier.cpp

```diff
@@ -417,6 +421,15 @@ class ShenandoahGenerationStatsClosure : public ShenandoahHeapRegionClosure {
               label, generation->name(), stats.regions(),
               byte_size_in_proper_unit(capacity), proper_unit_for_byte_size(capacity));
 
+    if (!generation->is_global()) {
+      // Humongous waste is currently only tracked for young and old generations
```

Comment:
Why?

(May be this gets back to my question about where and how this statistic is being used or being planned to be used.)

Reply:
I will try to change this. Ultimately, I want the "global" generation to subsume the usage metrics currently tracked in `ShenandoahHeap`.